### PR TITLE
Migrate BigQuery hotfix

### PR DESCRIPTION
**Pull request by robabram**
Updates to tests and mysql unittest helper.

### CODE REVIEW

File: rdr_service/api/dv_order_api.py [hunk 1]
```diff
@@ -114,9 +183,9 @@ def _put_supply_request(self, resource, bo_id):
             )
             p_id = from_client_participant_id(pid.value)
         except AttributeError as e:
-            raise BadRequest(e.message)
+            raise BadRequest(e)
```
Comment (Seeker1911) [inline rdr_service/api/dv_order_api.py]: I think we need str(e)

Comment (Seeker1911) [inline rdr_service/api/dv_order_api.py]:
```suggestion
            raise BadRequest(str(e))
```

File: rdr_service/api/dv_order_api.py [hunk 2]
```diff
@@ -114,9 +183,9 @@ def _put_supply_request(self, resource, bo_id):
...
         except Exception as e:
-            raise BadRequest(e.message)
+            raise BadRequest(e)
```
Comment (Seeker1911) [inline rdr_service/api/dv_order_api.py]:
```suggestion
            raise BadRequest(str(e))
```

File: rdr_service/api/dv_order_api.py [hunk 3]
```diff
@@ -125,47 +194,65 @@ def _put_supply_request(self, resource, bo_id):
...
-            tracking_status = fhir.extension.get(url=VIBRENT_FHIR_URL + "tracking-status").valueString
-            tracking_id = fhir.identifier.get(system=VIBRENT_FHIR_URL + "trackingId").value
+            tracking_status = fhir.extension.get(
+                url=VIBRENT_FHIR_URL + "tracking-status"
+            ).valueString
+            if tracking_status:
+                tracking_status = tracking_status.lower()
         except AttributeError as e:
-            raise BadRequest(e.message)
+            raise BadRequest(e)
```
Comment (Seeker1911) [inline rdr_service/api/dv_order_api.py]:
```suggestion
            raise BadRequest(str(e))
```

File: rdr_service/api/dv_order_api.py [hunk 4]
```diff
@@ -125,47 +194,65 @@ def _put_supply_request(self, resource, bo_id):
...
         except Exception as e:
-            raise BadRequest(e.message)
+            raise BadRequest(e)
```
Comment (Seeker1911) [inline rdr_service/api/dv_order_api.py]:
```suggestion
            raise BadRequest(str(e))
```

File: tests/test-data/dv_order_api_post_supply_delivery.json [hunk 5]
```diff
@@ -51,7 +51,7 @@
   "extension": [
     {
       "url": "http://joinallofus.org/fhir/tracking-status",
-      "valueString": "enroute"
+      "valueString": "in_transit"
```
Comment (Seeker1911) [inline tests/test-data/dv_order_api_post_supply_delivery.json]: perhaps we should make this upper case to ensure we're catching it correctly everywhere.

Comment (robabram) [reply tests/test-data/dv_order_api_post_supply_delivery.json]: PTSC is sending it to us lowercase, so I think we should probably keep it lowercase here.

**Pull request merged**